### ISSUE TITLE
boot parameter is silently ignored for Poisson glm

### ISSUE THREAD

**Issue by Ax3man** [lamho86/phylolm#13]
From the help, but with Poisson:

```
set.seed(123456)
tre = rtree(50)
x = rTrait(n=1,phy=tre)
X = cbind(rep(1,50),x)
y = rbinTrait(n=1,phy=tre, beta=c(-1,0.5), alpha=1 ,X=X)
dat = data.frame(trait01 = y, predictor = x)

fit = phyloglm(trait01~predictor,phy=tre,data=dat,boot=100, method = 'poisson_GEE')
summary(fit)
```

`phyloglm` doesn't do anything with the bootstrapping for Poisson models, but does run the model normally.  Asking for the `summary` then returns an error from `print.summary.phyloglm`:

```
Error in if (x$bootNrep > 0) cat("      Parametric bootstrap results based on",  : 
  argument is of length zero
```

presumably since it is looking for bootstrap results that aren't actually there.

**Comment by lamho86** [comment]
You are right. We haven't implemented bootstrap for Poisson models yet. I will do it soon.

**Comment by Ax3man** [comment]
Not a problem of course! Just mentioning it here since it is perhaps not so clear to users what is going on when this fails.

**Comment by Syrph** [comment]
Was bootstrapping ever done for these models? I'm using the bootstrap for normal phylolm and it'd be great to plot model results alongside with same method. Or is it easy to code up from inspecting the function?

**Comment by lamho86** [comment]
Coding bootstrap for Poisson glm is straightforward. All you need is a function to simulate from the Poisson model (similar to the rTrait/rbinTrait functions). Then, you can integrate it into phyloglm just like the boostraps for logistic regression. I've got a lot on my plate, so I couldn't do it right now. Please do a pull request if you code up something and would like to add them to our package.

**Comment by claireramsay** [comment]
Hi! was this function for bootstrapping with poisson GLM ever incorporated? Thanks so much!

**Comment by lamho86** [comment]
Hi. Unfortunately, I haven't done it yet. I will start working on it and hopefully get it done this summer.

**Comment by Rekyt** [comment]
Hi @lamho86 👋,
First of all thank you for building `phylolm`, which seems to solve many people's problems.
As a maintainer of scientific software myself, I fully understand the problem and time it takes to maintain and develop new features for a package, and I'm not asking here that you do rush it or prioritize over the rest of the work that you have to do.

I'm just writing this comment to ask @Ax3man, @ra-barber, @lauterbur, and @claireramsay, if any of you did implement bootstrapping by themselves in the end?
Because if it's the case, even a small script could really help the community here!
